### ISSUE TITLE
[Upsun] Upsun ify a Laravel app should invite users to add the Laravel bridge

### ISSUE THREAD

**Issue by thomasdiluccio** [platformsh/platformify#199]
upsun ify applied on a new Laravel app makes no mention of `platformsh/laravel-bridge`. It seems those lines are not hit:
https://github.com/platformsh/platformify/blob/19e15b3607a968d9f8bb333526537c4753be4453/platformifier/laravel.go#L41-L42

## How to reproduce

```
composer create-project laravel/laravel new-laravel-app

cd new-laravel-app

upsun ify
```

![image](https://github.com/platformsh/platformify/assets/3739767/512df792-faaa-4686-9904-1bf4ed8cb888)

**Comment by gilzow** [comment]
this appears to be an issue with `appRoot` at [line 31 of laravel.go](https://github.com/platformsh/platformify/blob/19e15b3607a968d9f8bb333526537c4753be4453/platformifier/laravel.go#L31)

What's odd is that ApplicationRoot.Ask determines the correct value:
```
ApplicationRoot.Ask called!
working directory is: /Users/gilzow/projects/delete/flyio-test-larave
path.dir is: 0x1098c00
Composer path is: /Users/gilzow/projects/delete/flyio-test-larave/composer.json
determined appRoot in application_root.go is: .
```
so we've determined the correct location at this stage. However, once we create `input` via `answers.ToUserInput()` in [platformify.go at line 66](https://github.com/platformsh/platformify/blob/19e15b3607a968d9f8bb333526537c4753be4453/commands/platformify.go#L66), `input.ApplicationRoot` is set back to `/`:
```
app root after creating input: /
```
## ah-ha!  found it!
@akalipetis  in [answer.go](https://github.com/platformsh/platformify/blob/19e15b3607a968d9f8bb333526537c4753be4453/internal/question/models/answer.go#L117), when converting the answers into a UserInput object (?), the return of `filepath.Join()` when `a.ApplicationRoot` is a `.` results in a return of `/` instead of `./` or just `.`.

Seems like what we want at this point is to have appRoot be a relative directory, correct? Shouldn't we prepend the `ApplicationRoot` property with a `.` at [line 117](https://github.com/platformsh/platformify/blob/19e15b3607a968d9f8bb333526537c4753be4453/internal/question/models/answer.go#L117)? ie
```
ApplicationRoot:    "." + filepath.Join(string(os.PathSeparator), a.ApplicationRoot),
```
When doing that, everything works:
```
working directory is: /Users/gilzow/projects/delete/flyio-test-larave
path.dir is: 0x1098c00
Composer path is: /Users/gilzow/projects/delete/flyio-test-larave/composer.json
determined appRoot in application_root.go is: .
inside of ToUserInput method of Answers
The current value of ApplicationRoot from the answers variable is: . 
The testPath when using Join: ./ 
app root after creating input: ./
application root is: ./
appPath: .
jsonPaths: [composer.json]

Please use composer to add the Laravel Bridge to your project:

    composer require platformsh/laravel-bridge

```
and even if i move composer.json into a subdirectory:
```
working directory is: /Users/gilzow/projects/delete/flyio-test-larave
path.dir is: 0x1098c00
Composer path is: /Users/gilzow/projects/delete/flyio-test-larave/foo/composer.json
determined appRoot in application_root.go is: foo

inside of ToUserInput method of Answers
The current value of ApplicationRoot from the answers variable is: foo 
The testPath when using Join: ./foo 
app root after creating input: ./foo
application root is: ./foo
appPath: foo
jsonPaths: [foo/composer.json]

Please use composer to add the Laravel Bridge to your project:

    composer require platformsh/laravel-bridge
```

**Comment by akalipetis** [comment]
Thanks a lot @gilzow for the help - the problem was indeed there, but the root cause was in the way we handle root directories in the `.Find()` command, which affected both Laravel and Django.

It should be fixed in #202
